awesimsoss/awesim.py

```diff
@@ -134,8 +134,8 @@ def __init__(self, ngrps, nints, star=None, planet=None, tmodel=None, snr=700,
         self.nints = nints
         self.nresets = nresets
         self.nframes = (self.nresets+self.ngrps)*self.nints
-        self.obs_date = datetime.datetime.now().strftime('%x')
-        self.obs_time = datetime.datetime.now().strftime('%X')
+        self.obs_date = datetime.datetime.now().strftime("%d/%m/%y")
```

Comment:
Actually, I think it needs to be `%Y-%m-%d`. Thanks for catching this!

Reply:
Actually... is it `%d/%m/%y`? Is this in the documentation or is it just the only way it runs through the pipeline?

Reply:
Oh, just saw #58. Ok, nevermind!

Reply:
Though the description the FITS file stores in the header reads "[yyyy-mm-dd] UTC date at start of exposure" so should it be `%Y-%m-%d`?

Reply:
I'm not sure, I'll run a test and see whether the pipeline complains if the format is ```%Y-%m-%d```. The pipeline definitely works with the ```%d/%m/%y``` format.

Reply:
Just tested: simulations are compatible with the stage 1 pipeline if the OBS_DATE header keyword is in ```%Y-%m-%d``` format as well. Given that the header description lists this format, its probably the best option.